### PR TITLE
hello from kone again

### PR DESCRIPTION
**Pull request by Laliska**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Misenko/ReviewNinja-Welcome/pull/3" target="_blank">ReviewNinja

File: hello [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello from konee!
```
Comment (Misenko) [inline hello]: fix it! !fix

Comment (Misenko) [inline hello]: fixed !fixed

Comment (Misenko) [inline hello]: !completed

**Pull request merged**